src/ActionUnit.h

```diff
@@ -45,7 +45,7 @@ class ActionUnit
     friend class XMLimport;
 
 public:
-                          ActionUnit( Host * pHost ) : mpHost(pHost), mMaxID(0) {;}
+                          ActionUnit( Host * pHost ) : mpHost(pHost), mMaxID(0), mpToolBar(), mpEasyButtonBar(), mModuleMember() {;}
```

Comment:
Does this form of initialisation ensure that:
+ ActionUnit::mpToolBar get a `nullptr`
+ ActionUnit::mpEasyButtonBar() get a `nullptr`
+ ActionUnit::mModuleMember get a `false`
initial value - or do we need to explicitly include them in the `()`

If we are skipping things that are NOT needed then I think the `;` in `{;}` shouldn't be there!